packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -460,6 +460,18 @@ export class Button extends BaseComponent<IButtonProps, {}> {
     protected _skipComponentRefResolution: boolean;
 }
 
+// @public (undocumented)
+export const ButtonGlobalClassNames: {

```

Comment:
Not sure we want these surfaced? At least if we decide to do it should be in a consistent manner for all the controls